### PR TITLE
Fix CMR action menu positioning + styling updates

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description
We had an issue with the action menu that was not properly positioning along the right side of the button. This PR fixes that issue, along with some styling updates to the linode action menu now that it's implemented in the table row.

## Type of Change
- Non breaking change ('update')

If the any above types of change apply to this pull request, please ensure to include one of the listed keywords in the pull request title.

## Applicable E2E Tests

To run relevant E2E tests, run these commands in 3 separate terminals:

1. `yarn && yarn start`
2. `yarn selenium`
3. `yarn e2e --spec=COMMA_SEPARATED_LIST_OF_SPECS_HERE --browser=headlessChrome`

## Note to Reviewers

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

### CODE REVIEW

File: packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
   MenuPopover
 } from '@reach/menu-button';
 import '@reach/menu-button/styles.css';
+// import Popover, { positionDefault, positionRight } from '@reach/popover';
```
Comment (WilkinsKa1) [inline packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx:11]: Will remove this pre-merge so long as it's g2g.

File: packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx [hunk 2]
```diff
@@ -148,11 +147,12 @@ const ActionMenu: React.FC<CombinedProps> = props => {
           aria-label={ariaLabel}
           onMouseDown={handleClick}
           onKeyDown={handleKeyPress}
+          ref={ref}
         >
           <MoreHoriz aria-hidden type="primary" />
           {inlineLabel && <p className={classes.buttonLabel}>{inlineLabel}</p>}
         </MenuButton>
-        <MenuPopover className={classes.popover} portal={true}>
+        <MenuPopover className={classes.popover}>
```
Comment (WilkinsKa1) [inline packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx:155]: portal defaults to true, so removed as redundant.

File: packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx [hunk 3]
```diff
@@ -118,6 +115,8 @@ const ActionMenu: React.FC<CombinedProps> = props => {
 
   const [actions, setActions] = React.useState<Action[]>([]);
 
+  const ref = React.useRef<HTMLButtonElement>(null);
```
Comment (WilkinsKa1) [inline packages/manager/src/components/ActionMenu_CMR/ActionMenu_CMR.tsx:118]: Pretty sure I can remove the ref? I used this initially when I was testing with popover but I don't think we need it anymore.

**Comment by WilkinsKa1** [comment]
@DevDW thanks for taking a look! I was able to fix the border radius issue (when the menu displays below the button) along with some cleanup. I will hold off on the other styling piece you mentioned since you are addressing that separately- when they're both merged we can confirm alignment with Jay.

File: packages/manager/public/mockServiceWorker.js [hunk 4]
```diff
@@ -7,80 +7,80 @@
 /* eslint-disable */
 /* tslint:disable */
 
-const INTEGRITY_CHECKSUM = '3b8ed9e2e9775de6221fe86bf5bbc2ec'
-const bypassHeaderName = 'x-msw-bypass'
+const INTEGRITY_CHECKSUM = '3b8ed9e2e9775de6221fe86bf5bbc2ec';
+const bypassHeaderName = 'x-msw-bypass';
```
Comment (WilkinsKa1) [inline packages/manager/public/mockServiceWorker.js:11]: I didn't actually make any changes here, I think with a recent pull my format on save config made these changes.

**Comment by tiffwong** [comment]
> Menu is much better. Don't know if it was intentional but the hover states on the inline actions are different now:
> 
> Screen Shot 2020-07-08 at 7 40 52 AM
> 
> Screen Shot 2020-07-08 at 7 40 58 AM

@Jskobos I one of my PR's will fix this

**Comment by WilkinsKa1** [comment]
@Jskobos @tiffwong thanks for reviewing! I made updates for dark mode and fixed the padding on the button. I couldn't replicate the table gaping issue, though I did just pull latest develop here so that probably fixed it for me.

**Comment by WilkinsKa1** [comment]
@johnwcallahan @tiffwong I removed the border radius, this fixes both the issues mentioned!

**Pull request merged**